src/adapters/baseAdapter..ts

```diff
@@ -0,0 +1,15 @@
+import requestManager, { RequestParamsType } from 'lib/requestManager';
+
+class BaseAdapter {
+  static baseUrl = process.env.REACT_APP_API_ENDPOINT;
+
+  getRequest(endpoint: string, params: RequestParamsType) {
+    return requestManager('GET', `${BaseAdapter.baseUrl}/${endpoint}`, params);
+  }
+
+  postRequest(endpoint: string, params: RequestParamsType) {
```

Comment:
Should this be request option to be more accurate with the request manager?
```suggestion
  postRequest(endpoint: string, requestOption: RequestParamsType) {
```

Reply:
I used the naming conventions from `Compass` so I think should be default, but I can go with this as well